packages/client/src/products/__fixtures__/getProductGrouping.fixtures.ts

```diff
@@ -2,10 +2,8 @@ import { rest, RestHandler } from 'msw';
 import type { ProductGrouping } from '../types';
 
 const path = '/api/commerce/v1/products/:id/grouping';
-/**
- * Response payloads.
- */
-export default {
+
+const fixtures = {
   success: (response: ProductGrouping): RestHandler =>
     rest.get(path, async (req, res, ctx) =>
```

Comment:
Missing underscore on first parameter
```suggestion
    rest.get(path, async (_req, res, ctx) =>
```

Reply:
should be ok now. thanks 👍